ui/appui-layout-react/src/appui-layout-react/widget/tools/button/Button.tsx

```diff
@@ -43,7 +43,7 @@ export class ToolbarButton extends React.PureComponent<ToolbarButtonProps> {
     };
 
     if (this.props.small) {
-      let backgroundOpacity = TOOLBAR_OPACITY_DEFAULT;
+      let backgroundOpacity = Number(document.documentElement.style.getPropertyValue("--buic-toolbar-opacity"));
```

Comment:
After looking at the whole code, why don't we just apply `background-opacity: var("--buic-toolbar-opacity");` to the `nz-toolbar-button-button` class in CSS (or add a `nz-small` on top if this really only apply on `small={true}` buttons ? Rather than using the style attribute on it ? (All the settings would be at the same place ?)

Reply:
Because we still have all the proximity code in 3.x. I'd like to remove all that code in 4.0, but I wanted this change to be as small as possible for 3.x.